test/Integration/broadcast-2d.mlir

```diff
@@ -0,0 +1,27 @@
+// RUN: tpp-run --vector-to-XSMM %s -e columnBroadcast --entry-point-result=void -print --seed 123 2>&1 | FileCheck %s -check-prefix=COLUMNBROADCAST
+// RUN: tpp-run --linalg-to-loops %s -e columnBroadcast --entry-point-result=void -print -seed 123 2>&1 | FileCheck %s -check-prefix=COLUMNBROADCAST
```

Comment:
Same as transpose, you have no idea if your path is being selected for this pattern.

Reply:
I do actually, this was not supported earlier, it is now in the vector-to-xsmm lowering path.

Reply:
In order to ensure correctness, I'll add tpp-opt -default-tpp-pipeline="--vector-to-xsmm" run line to print vector output and verify that vector output is indeed getting generated, does that sound reasonable? I could try printing from tpp-run -print-after but I don't know how to disable multithreading from commandline.